code-scanning/black-duck-security-scan-ci.yml

```diff
@@ -0,0 +1,34 @@
+name: CI-Black-Duck-Security-Scan
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+    
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v3
+      - name: Black Duck SCA Scan
+        uses: blackduck-inc/black-duck-security-scan@v2.0.0         
```

Comment:
is there a way we can make the version dynamic. Also check what is the standard used by other action starter workflows

Reply:
Hello community

Reply:
> is there a way we can make the version dynamic. Also check what is the standard used by other action starter workflows

Changed tag name to tag commit hash

Reply:
Hello